src/commands/synthetics/reporters/default.ts

```diff
@@ -63,6 +63,17 @@ const renderStep = (step: Step) => {
   return `    ${icon} | ${duration} - ${step.description}${value}${error}`
 }
 
+const renderSkippedSteps = (steps: Step[]): string | undefined => {
+  if (!steps.length) {
+    return
+  }
+  if (steps.length === 1) {
+    return renderStep(steps[0])
```

Comment:
What is the purpose of rendering a full "skipped step" when there is (I think?) no information available anyway (no duration, no error...)? 🤔

Reply:
It contains the name of the step, and since it take as much space might as well show it I guess